src/modals/WelcomeModal.tsx

```diff
@@ -64,98 +51,38 @@ const Welcome = (props: PropsWelcomeModal) => {
-                                        </IonSelectOption>
-                                    ))}
-                                </IonSelect>
-                            </IonItem>
-                        </IonList>
-                    </IonCardContent>
-                    <IonCol>
-                        or
-                    </IonCol>
-                    <IonCol>
-                        <IonButton
-                            color="dark-basic"
-                            onClick={() => {
-                                importConfig()
-                                    .then((config) => {
-                                        storage.set.cycles(config.cycles)
-                                            .then(() => {
-                                                updateCycles(config.cycles);
-                                                    confirmAlert({
-                                                        header: "Configuration has been imported",
-                                                        cssClass: "header-color",
-                                                        buttons: [{
-                                                            text: "OK",
-                                                            role: "confirm",
-                                                            handler: () => {
-                                                                props.setIsOpen(false);
-                                                            },
-                                                        }],
-                                                    });
-                                            })
-                                            .catch((err) => {
-                                                console.error(err);
-                                            });
-                                    })
-                                    .catch((err) => {
-                                        console.error(err);
-                                    });
-                            }}
-                        >
-                            <IonIcon slot="start" icon={cloudDownloadOutline} />
-                            Import
-                        </IonButton>
-                    </IonCol>
-                </IonCard>
+            <IonContent fullscreen color="basic">
+                <IonCol>
+                    <IonLabel class="welcome" color="dark-basic">
+                        Please mark the days of your last period.
+                    </IonLabel>
+                </IonCol>
+                <IonDatetime
+                    class="welcome"
+                    ref={refDatetime}
+                    color="basic"
+                    presentation="date"
+                    locale="en-GB"
+                    size="cover"
+                    multiple
+                    firstDayOfWeek={1}
+                >
```

Comment:
You can write

```jsx
<IonDatetime
                    class="welcome"
                    ref={refDatetime}
                    color="basic"
                    presentation="date"
                    locale="en-GB"
                    size="cover"
                    multiple
                    firstDayOfWeek={1}
                />
```

instead of

```jsx
<IonDatetime
                    class="welcome"
                    ref={refDatetime}
                    color="basic"
                    presentation="date"
                    locale="en-GB"
                    size="cover"
                    multiple
                    firstDayOfWeek={1}
                >
                </IonDatetime>
```

Reply:
Fixed